src/components/JiscCard/JiscCard.stories.js

```diff
@@ -59,24 +59,110 @@ export const CardStyle2 = () => (
 
 export const CardStyle3 = () => (
     <JiscCard cardCategory='style3'>
-        <CardMedia
-            component='img'
-            height='140px'
-            image='https://images.unsplash.com/photo-1470938330186-da818256fa34?ixid=MXwxMjA3fDB8MHxzZWFyY2h8NXx8YmVlc3xlbnwwfHwwfA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60'
-        />
+        <CardActionArea>
+            <CardMedia
+                component='img'
+                height='140px'
+                image='https://images.unsplash.com/photo-1470938330186-da818256fa34?ixid=MXwxMjA3fDB8MHxzZWFyY2h8NXx8YmVlc3xlbnwwfHwwfA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60'
+            />
 
-        <CardContent>
-            <Typography gutterBottom variant='h5' component='h2'>
-                Lizard
-            </Typography>
-            <Typography variant='body2' color='textSecondary' component='p'>
-                Lizards are a widespread group of squamate reptiles, with over 6,000 species, ranging across all
-                continents except Antarctica
-            </Typography>
-        </CardContent>
+            <CardContent>
+                <Typography gutterBottom variant='h5' component='h2'>
+                    Lizard
+                </Typography>
+                <Typography variant='body2' color='textSecondary' component='p'>
+                    Lizards are a widespread group of squamate reptiles, with over 6,000 species, ranging across all
+                    continents except Antarctica
+                </Typography>
+            </CardContent>
 
-        <CardActions>
-            <Button size='small'>Learn More</Button>
-        </CardActions>
+            <CardActions>
+                <Button size='small'>Learn More</Button>
+            </CardActions>
+        </CardActionArea>
```

Comment:
I think the `CardActionArea` should end before the `CardActions` rather than have the `CardActions` contained.

The markup is a `button` within a `button` which is not valid

Reply:
I don't think the CardActions should be used when using the CardActionArea. I'll remove from the example.